### PR TITLE
[New Rule] Attempt to Unload Elastic Endpoint Security Kernel Extension

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/806

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/macos/defense_evasion_unload_endpointsecurity_kext.toml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+[metadata]
+creation_date = "2020/01/05"
+maturity = "production"
+updated_date = "2020/01/05"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies attempt to unload the Elastic Endpoint Security kernel extension via kextunload command.
```
Comment (bm11100) [inline rules/macos/defense_evasion_unload_endpointsecurity_kext.toml:9]:
```suggestion
Identifies attempts to unload the Elastic Endpoint Security kernel extension via the kextunload command.
```

File: rules/macos/defense_evasion_unload_endpointsecurity_kext.toml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+"""
+from = "now-9m"
+index = ["auditbeat-*", "logs-endpoint.events.*"]
+language = "kuery"
+license = "Elastic License"
+name = "Attempt to Unload Elastic Endpoint Security Kernel Extension"
+risk_score = 43
```
Comment (bm11100) [inline rules/macos/defense_evasion_unload_endpointsecurity_kext.toml:16]:
```suggestion
risk_score = 73
```

File: rules/macos/defense_evasion_unload_endpointsecurity_kext.toml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+rule_id = "70fa1af4-27fd-4f26-bd03-50b6af6b9e24"
+severity = "medium"
```
Comment (bm11100) [inline rules/macos/defense_evasion_unload_endpointsecurity_kext.toml:18]:
```suggestion
severity = "high"
```

**Review by bm11100 — APPROVED** [review]

**Review by brokensound77 — APPROVED** [review]
LGTM 👍

**Pull request merged**